scripts/pup

```diff
@@ -126,4 +126,8 @@ if [[ "$1" == "post-deploy" ]] && ([[ "$2" == "-a" ]] || [[ "$2" == "--all" ]] |
     exit
 fi
 
```

Comment:
- Need to prompt user if directory is found under srv
- Need to create standalone logs on disk
- be able to run under pup utils non-root. allow additional inputs